packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -363,9 +361,14 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
     return false;
   }
 
-  setRef = (elem: React.ReactInstance): void => {
-    // findDOMNode needed to get correct DOM ref with react-hot-loader, see https://github.com/gaearon/react-hot-loader/issues/964
-    this._root.current = ReactDOM.findDOMNode(elem) as HTMLElement;
+  setRef = (elem: HTMLElement): void => {
+    this._root.current = elem;
+    handleRef(this.props.innerRef, elem);
+    if (process.env.NODE_ENV !== 'production') {
+      if (elem !== null && !(elem?.nodeType === 1)) {
+        throw new Error('Please, use HTMLElement');
```

Comment:
```suggestion
        throw new Error('FocusZone: we expect that ElementType ("as" prop) will be a plain element (div, span, etc.) or an element that supports ref forwarding (React.forwardRef())');
```

Let's make this error more helpful